docs/guides/editor_features/watching.md

```diff
@@ -11,15 +11,17 @@ have the changes automatically reflected in your browser.
     For better performance, install [watchdog](https://pypi.org/project/watchdog/).
     Without watchdog, marimo resorts to polling.
 
-## marimo's file format
+## marimo's File Format
```

Comment:
We use sentence casing not title casing for headings. Here and elsewhere. For example

<img width="250" alt="image" src="https://github.com/user-attachments/assets/b17a2d0f-1e60-46c4-9b59-32b5031114ed" />